cmd/determinize-prow-config/main.go

```diff
@@ -65,9 +65,7 @@ func main() {
 
 type determinizeProwConfigFunctors struct{}
 
-func (d determinizeProwConfigFunctors) ModifyQuery(queryCopy *prowconfig.TideQuery, repo string) {
-	queryCopy.Orgs = nil
-	queryCopy.Repos = []string{repo}
+func (d determinizeProwConfigFunctors) ModifyQuery(*prowconfig.TideQuery, string) {
```

Comment:
Let's add a `panic(fmt.Errorf("not implemented"))`

Reply:
Nope, should be that way, it's empty for a reason. This one is not using it at all but every others have implementation here.

Why I moved:
```
queryCopy.Orgs = nil
queryCopy.Repos = []string{repo}```

out of this place? Because in every implementation of that function I was doing that, so that was a redundancy. Original code had this inside and it was my mistake to take it out of it.